boards/arm/nrf5340pdk_nrf5340/nrf5340pdk_nrf5340_cpuapp_common.dts

```diff
@@ -164,21 +164,23 @@
 	};
 };
 
-&sram0 {
+/ {
 
-	partitions {
-		compatible = "fixed-partitions";
+	reserved-memory {
 		#address-cells = <1>;
 		#size-cells = <1>;
+		ranges;
+
+		sram0_image: image@20000000 {
```

Comment:
Is it OK that this has the same unit address as image_s?

Reply:
Yes, this is intended  @mbolivar-nordic , this not represents the union of image_s and image_ns

Reply:
Doesn't this generate warnings in some dtc / edtlib configurations? Do we have those disabled for our ICs?

Reply:
No it does not , I guess because the prefix names differ in each node